### PR TITLE
chore: fix error when data tobe 0 null

### PR DESCRIPTION
**Pull request by anc95**
if the cache data is 0 or null, now the initialData will be from config. it is not expected

### CODE REVIEW

File: src/use-swr.ts [hunk 1]
```diff
@@ -235,7 +235,7 @@ function useSWR<Data = any, Error = any>(
     fn = config.fetcher
   }
 
-  const initialData = cache.get(key) || config.initialData
+  const initialData = cache.get(key) === undefined ? config.initialData : cache.get(key)
```
Comment (shuding) [inline src/use-swr.ts:238]:
```suggestion
  const cachedData = cache.get(key)
  const initialData = typeof cachedData === 'undefined' ? config.initialData : cachedData
```

File: src/use-swr.ts [hunk 2]
```diff
@@ -479,7 +479,7 @@ function useSWR<Data = any, Error = any>(
     // and trigger a revalidation
 
     const currentHookData = stateRef.current.data
-    const latestKeyedData = cache.get(key) || config.initialData
+    const latestKeyedData = cache.get(key) === undefined ? config.initialData : cache.get(key)
```
Comment (shuding) [inline src/use-swr.ts:482]:
```suggestion
    const cachedData = cache.get(key)
    const latestKeyedData = typeof cachedData === 'undefined' ? config.initialData : cachedData
```

**Comment by anc95** [comment]
I just updated code as your suggest

**Review by shuding — APPROVED** [review]
Thanks!

**Pull request merged**